dev/install-common-deps.sh

```diff
@@ -43,14 +43,14 @@ if [[ "$INSTALL_SMALL_PYTHON_DEPS" == "true" ]]; then
   # When downloading large packages from PyPI, the connection is sometimes aborted by the
   # remote host. See https://github.com/pypa/pip/issues/8510.
   # As a workaround, we retry installation of large packages.
-  retry-with-backoff pip install --quiet -r ./dev/small-requirements.txt
+  retry-with-backoff pip install -r ./dev/small-requirements.txt
```

Comment:
Removed `--quiet` because this makes it difficlt to debug package installation.

Reply:
Without `--quiet`, pip install would generates logs like:

```
Requirement already satisfied: scipy in /usr/share/miniconda/envs/test-environment/lib/python3.6/site-packages (from -r ./dev/small-requirements.txt (line 2)) (1.5.4)
Requirement already satisfied: sqlalchemy>=1.3.21 in /usr/share/miniconda/envs/test-environment/lib/python3.6/site-packages (from -r ./dev/small-requirements.txt (line 5)) (1.4.15)
Collecting pytest==3.2.1
  Downloading pytest-3.2.1-py2.py3-none-any.whl (186 kB)
Collecting pytest-cov==2.6.0
  Downloading pytest_cov-2.6.0-py2.py3-none-any.whl (14 kB)
Collecting pytest-localserver==0.5.0
  Downloading pytest-localserver-0.5.0.tar.gz (20 kB)
Collecting moto!=2.0.7
  Downloading moto-2.0.6-py2.py3-none-any.whl (1.2 MB)
Collecting azure-storage-blob>=12.0.0
  Downloading azure_storage_blob-12.8.1-py2.py3-none-any.whl (345 kB)
Collecting py>=1.4.33
  Downloading py-1.10.0-py2.py3-none-any.whl (97 kB)
Requirement already satisfied: setuptools in /usr/share/miniconda/envs/test-environment/lib/python3.6/site-packages (from pytest==3.2.1->-r ./dev/small-requirements.txt (line 7)) (52.0.0.post20210125)
```

This allows us to understand what installs what.

Reply:
Awesome :)